### PR TITLE
Content: Add + Fix Leader SC Ceremonies + Loner Names

### PR DESCRIPTION
**Pull request by izzyphobic**
## About The Pull Request

Fixes some of the leader ceremonies that were supposed to be generating for a certain Starclan trait instead generating for a leader trait (a SC cat that had the trait Calm could generate a ceremony as though they were Sneaky), plus adds some NEW ceremonies and 100+ new loner names (organized the loner name list a little.)

## Why This Is Good For ClanGen

I had heard some new Leader Ceremonies were needed for some traits, so I got to work on some! Additionally, it was noticed that in a previous merge there may have been a misunderstanding in how the Leader Ceremony traits work, so every cat should now instead get a ceremony that actually matches the trait they have.

https://discord.com/channels/1003759225522110524/1101276997751164948/1281474120889929821

## Proof of Testing

image
image
image
image

### CODE REVIEW

File: resources/dicts/lead_ceremony_sc.json [hunk 1]
```diff
@@ -237,6 +250,19 @@
       "lead_trait": ["calm"],
       "star_trait": [],
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "m_c, recognized for {PRONOUN/m_c/poss} steady composure, greeted r_c with a courteous nod as {PRONOUN/m_c/subject} approached. Approval alight in {PRONOUN/r_c/poss} expression, r_c bestows upon  m_c a life for [virtue], and as it settled within {PRONOUN/m_c/object}, {PRONOUN/m_c/subject} could feel a subtle, yet powerful shift within, one of fiery initiative.",
```
Comment (scribblecrumb) [inline resources/dicts/lead_ceremony_sc.json]:
```suggestion
          "text": "m_c, recognized for {PRONOUN/m_c/poss} steady composure, greeted r_c with a courteous nod as {PRONOUN/r_c/subject} approached. Approval alight in {PRONOUN/r_c/poss} expression, r_c bestows upon  m_c a life for [virtue]. As it settled within {PRONOUN/m_c/object}, {PRONOUN/m_c/subject} could feel a subtle, yet powerful shift within, one of fiery initiative.",
```
pronoun tag seemed to be directed toward the wrong cat, and a little sentence break to help with readability

File: resources/dicts/lead_ceremony_sc.json [hunk 2]
```diff
@@ -309,6 +335,19 @@
       "lead_trait": ["cold"],
       "star_trait": [],
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy"],
+      "life_giving": [
+        {
+          "text": "r_c approaches m_c with narrowed eyes, slightly unsettled by {PRONOUN/m_c/poss} aloofness. m_c is startled by the radiant warmth that surged through {PRONOUN/m_c/poss} chest as {PRONOUN/m_c/object} {VERB/r_c/are/is} granted a life for [virtue], and for the first time in a long while, a soft smile softens {PRONOUN/m_c/poss} cold demeanor.",
```
Comment (scribblecrumb) [inline resources/dicts/lead_ceremony_sc.json]:
```suggestion
          "text": "r_c approaches m_c with narrowed eyes, slightly unsettled by {PRONOUN/m_c/poss} aloofness. m_c is startled by the radiant warmth that surges through {PRONOUN/m_c/poss} chest as {PRONOUN/m_c/object} {VERB/r_c/are/is} granted a life for [virtue], and for the first time in a long while, a soft smile softens {PRONOUN/m_c/poss} cold demeanor.",
```
tense fix

File: resources/dicts/lead_ceremony_sc.json [hunk 3]
```diff
@@ -410,32 +458,37 @@
...
   
-    "lonesome_warrior_EXAMPLE": {
+    "lonesome_warrior": {
       "tags": [],
       "lead_trait": ["lonesome"],
       "star_trait": [],
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy"],
+      "life_giving": [
+        {
+          "text": "r_c gazed at the lonesome leader with a deep sympathy as {PRONOUN/r_c/subject} approached, sensing the weight of {PRONOUN/m_c/poss} solitude. m_c's eyes begin to well up as a life for [virtue] flows through {PRONOUN/m_c/object}, a warm embrace that warded off the cold isolation, even if for a brief moment.",
```
Comment (scribblecrumb) [inline resources/dicts/lead_ceremony_sc.json]:
```suggestion
          "text": "r_c gazes at the lonesome leader with a deep sympathy as {PRONOUN/r_c/subject} approach, sensing the weight of {PRONOUN/m_c/poss} solitude. m_c's eyes begin to well up as a life for [virtue] flows through {PRONOUN/m_c/object}, a warm embrace that wards off the cold isolation, even if for a brief moment.",
```

File: resources/dicts/lead_ceremony_sc.json [hunk 4]
```diff
@@ -452,27 +505,27 @@
...
-      "lead_trait": ["loving"],
-      "star_trait": [],
+      "lead_trait": [],
+      "star_trait": ["loving"],
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy"],
       "life_giving": [
         {
-          "text": "r_c walks up to m_c, offering a life for [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/smile/smiles}, and {VERB/r_c/state/states} that the Clan will do well under m_c's leadership.",
-          "virtue": ["acceptance", "bravery", "certainty", "clear judgment", "confidence", "courage", "determination", "endurance", "sympathy", "farsightedness", "friendship", "instincts", "mercy", "strength", "unity"]
+          "text": "m_c's ears perk up as {PRONOUN/m_c/subject} {VERB/m_c/begin/begins} to hear a deep, resonant rumbling. Reciprocating the intense purr, m_c gently bumps heads with r_c, a warrior known for the endless love {PRONOUN/m_c/subject} {VERB/r_c/hold/holds} for {PRONOUN/r_c/poss} Clan. A warm, sweet feeling cascades over m_c as {PRONOUN/m_c/subject} {VERB/m_c/receive/receives} a life for [virtue].",
```
Comment (scribblecrumb) [inline resources/dicts/lead_ceremony_sc.json]:
```suggestion
          "text": "m_c's ears perk up as {PRONOUN/m_c/subject} {VERB/m_c/begin/begins} to hear a deep, resonant rumbling. Reciprocating the intense purr, m_c gently bumps heads with r_c, a warrior known for the endless love {PRONOUN/r_c/subject} {VERB/r_c/hold/holds} for {PRONOUN/r_c/poss} Clan. A warm, sweet feeling cascades over m_c as {PRONOUN/m_c/subject} {VERB/m_c/receive/receives} a life for [virtue].",
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Comment by izzyphobic** [comment]
Committed your changes, should be okay now, let me know if there is anything else :3

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**